mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -1,15 +1,57 @@
 CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 	CLASS class_1012 Format
 		FIELD field_4993 pixelDataFormat I
-		FIELD field_4994 bytesPerPixel I
+		FIELD field_4994 totalChannels I
```

Comment:
Are you sure?

1. If I remember correctly, it's passed to a GL function whose parameter is called "bytes per pixel" (which is not necessarily the number of channels, a channel [is not necessarily one byte] (https://en.wikipedia.org/wiki/Color_depth#Deep_color_(30/36/48-bit))). It also doesn't really make sense to have an int for the channel count since it can only really be 1 or 3, so a `color` boolean would be more likely for what this is used for. 
2. What does "total" mean here?
3. It should be `channelCount` (or `totalChannelCount`), it's just the channel count, not the channels themselves.

Reply:
1. It's passed to a method where the parameter is simply named "channels" and has a description saying it is the number of channels to be used in that method.
2. Total is the total number of channels the format supports. 
3. I can make it `channelCount`. `totalChannelCount` is too much.

Reply:
> the total number of channels the format supports

Still not sure what "total" means here. The total of what? Is there also a non-total channel count...?

> I can make it channelCount

Yes, that's better.